### PR TITLE
Fix documentation comments

### PR DESCRIPTION
**Pull request by JosefPihrt**
This PR uses Roslynator analyzers to apply following code fixes:

* Add paragraphs to documentation comments ([RCS1226](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1226.md))
* Remove unused elements in documentation comments (RCS1228)
* Order elements in documentation comments (RCS1232)
* Remove duplicate word in documentation comments (RCS1243)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/roslyn-analyzers?pullRequest=4059) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/roslyn-analyzers?pullRequest=4059) before we can accept your contribution.<br/><br/>:x: JosefPihrt [sign now](https://cla.dotnetfoundation.org/dotnet/roslyn-analyzers?pullRequest=4059)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/roslyn-analyzers?pullRequest=4059) it.</sub>

File: src/Roslyn.Diagnostics.Analyzers/Core/ImportingConstructorShouldBeObsolete.cs [hunk 1]
```diff
@@ -11,11 +11,12 @@
 namespace Roslyn.Diagnostics.Analyzers
 {
     /// <summary>
-    /// The importing constructor for a MEF-exported type should be marked obsolete.
-    ///
+    /// <para>The importing constructor for a MEF-exported type should be marked obsolete.</para>
+    /// <para>
     /// <code>
```
Comment (sharwell) [inline src/Roslyn.Diagnostics.Analyzers/Core/ImportingConstructorShouldBeObsolete.cs:16]: ⚠️ The `<code>` element should not be placed within a `<para>` element. If this change was automated, it is likely a bug in the tool that made the change.

File: src/Utilities/Compiler/Options/AggregateCategorizedAnalyzerConfigOptions.cs [hunk 2]
```diff
@@ -15,14 +15,14 @@ namespace Analyzer.Utilities
     using static CategorizedAnalyzerConfigOptionsExtensions;
 
     /// <summary>
-    /// Aggregate analyzer configuration options:
-    ///
+    /// <para>Aggregate analyzer configuration options:</para>
+    /// <para>
     /// <list type="number">
```
Comment (sharwell) [inline src/Utilities/Compiler/Options/AggregateCategorizedAnalyzerConfigOptions.cs:20]: ⚠️ `<list>` should not be placed within `<para>`. If this change was automated, it is likely a bug in the tool.

File: src/Utilities/Compiler/Options/AggregateCategorizedAnalyzerConfigOptions.cs [hunk 3]
```diff
@@ -15,14 +15,14 @@ namespace Analyzer.Utilities
...
     /// <item><description>Per syntax tree options from <see cref="AnalyzerConfigOptionsProvider"/>.</description></item>
     /// <item><description>Options from an <strong>.editorconfig</strong> file passed in as an additional file (back compat).</description></item>
     /// </list>
-    ///
-    /// <inheritdoc cref="ICategorizedAnalyzerConfigOptions"/>
+    /// </para>
+    /// <para><inheritdoc cref="ICategorizedAnalyzerConfigOptions"/></para>
```
Comment (sharwell) [inline src/Utilities/Compiler/Options/AggregateCategorizedAnalyzerConfigOptions.cs:25]: ⚠️ `<inheritdoc>` is _allowed_ to be placed within `<para>`, but it changes the semantics. In this case, it should not have been changed.

**Comment by Evangelink** [comment]
@JosefPihrt Would you like to address the comments on this PR?

**Pull request closed without merging**